model/registry/gen-ai.yaml

```diff
@@ -14,6 +14,34 @@ groups:
               stability: experimental
               value: "openai"
               brief: 'OpenAI'
+            - id: microsoft
+              stability: experimental
+              value: "microsoft"
+              brief: 'Microsoft'
+            - id: huggingface
+              stability: experimental
+              value: "huggingface"
+              brief: 'Hugging Face'
+            - id: google
```

Comment:
Same here - VertexAI?

Reply:
VertexAI is a machine learning platform for building and using Gen AI.